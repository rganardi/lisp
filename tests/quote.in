(quote a)
(quote b)
(quote hi there)
(quote (quote hi))
(quote (lambda (x) hi))
(quote nil)
(quote ((lambda (x) x) (quote y)))
