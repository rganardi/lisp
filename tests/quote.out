a
b
hi there
(quote hi)
(lambda (x) hi)
nil
((lambda (x) x) (quote y))
